src/main/java/org/example/homework5/RandomNumberSum.java

```diff
@@ -0,0 +1,28 @@
+package org.example.homework5;
+import java.util.Random;
+
+public class RandomNumberSum {
+    public static void main(String[] args) {
+        Random random = new Random();
+        int[] occurrences = new int[121111111];
+        int sum = 0;
+        int count = 0;
+
+        while (true) {
+            int randomNumber = random.nextInt(100) + 1;
+            count++;
+            System.out.println("Generated number: " + randomNumber);
+
+            if (occurrences[randomNumber] == 1) {
```

Comment:
we discussed that we should exit the loop after we processed the number even if it appeared for the second time